README.rst

```diff
@@ -4,19 +4,27 @@ SkyPy: A package for modelling the Universe
 
 |Read the Docs| |GitHub| |Codecov|
 
-This package contains methods for modelling the Universe, galaxies and the Milky
-Way. Also included are methods for generating observed data.
+The SkyPy package contains methods for simulating populations of astronomical
```

Comment:
If people are looking for a one sentence description of SkyPy, this first sentence is usually the reference. I am not sure the current sentence is concise enough (and catchy/attractive) to serve this purpose.

Reply:
Agree with Simon. I would potentially keep the more impactful "This package contains methods for modelling the Universe, galaxies and the Milky Way." and then go on with the details in what this PR proposes.

Reply:
Okay, I put it back.